hasher-matcher-actioner/src/OpenMediaMatch/storage/interface.py

```diff
@@ -409,8 +410,19 @@ def bank_delete(self, name: str) -> None:
 
     # Bank content
     @abc.abstractmethod
-    def bank_content_get(self, id: t.Iterable[int]) -> t.Sequence[BankContentConfig]:
-        """Get the content config for a bank"""
+    def bank_content_get(
+        self, id: t.Iterable[int], signal_type: t.Optional[str] = None
+    ) -> t.Sequence[BankContentConfig]:
+        """
+        Get the content config for a bank.
+
+        Args:
+            id: The IDs of the bank content to retrieve
+            signal_type: Optional signal type to include in the response
+
+        Returns:
+            List of bank content configs, optionally including signal values
+        """
 
```

Comment:
unsure: As an alternative, we could provide a method that allows fetching the signals one at a time directly. The upsides would be that it's simpler to implement multiple interfaces, and would allow fetching strictly only the signals.